aca_required_temp.py

```diff
@@ -0,0 +1,189 @@
+
+import numpy as np
+
+import agasc
+from Chandra.Time import DateTime
+from Ska.Sun import nominal_roll, pitch
+from Ska.quatutil import radec2yagzag
+from Quaternion import Quat
+import chandra_aca
+from starcheck.star_probs import t_ccd_warm_limit
+from astropy.table import Table
+
+N_ACQ_STARS = 5
+EDGE_DIST = 30
+COLD_T_CCD = -21
+WARM_T_CCD = -5
```

Comment:
Open question about reasonable value for this to start: https://github.com/sot/aca_required_temp/commit/16bcfc3542c0ec7788d335879606d0e39f1e41c9